engine/src/test/java/com/datatorrent/stram/engine/StreamingContainerTest.java

```diff
@@ -94,4 +127,14 @@ public void emitTuples()
     private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
   }
 
+  private static class CommitAwareProcessOperator extends CommitAwareOperator
```

Comment:
This Operator has an input port. In the testOiOCommitted we need to create a logical plan with multiple operators requiring the output port of upstream op to be connected to the input port of the downstream op. If there is a single operator having both the ports then for the upstream operator the input port will not be connected and an error stating the same will be thrown. Hence, the  need for the second operator.